def raise_exception(e: Exception):
    raise e